decidim-core/app/assets/javascripts/decidim/select2.js.es6

```diff
@@ -6,3 +6,4 @@
 // = require select2_locale_fr
 // = require select2_locale_it
 // = require select2_locale_nl
+// = require select2_locale_pt
```

Comment:
We probably lack lots of other languages here, don't we?

Reply:
It seems to be missing `_uk` and `_ru`, yeah.

Reply:
Thank you everybody, I have made a PR to add the missing locales.